fix(html): Specify the default layout for heading elements

Arisch24 opened the pull request:
Describe that all heading elements are block-level by default.

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

Add description for headings which are displayed as block-level

### Motivation

Better description for readers

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

Fixes #28475
<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

Arisch24 committed:
Fix #28475

Describe that all heading elements are block-level by default.

dipikabh commented on files/en-us/web/html/element/heading_elements/index.md:
```diff
@@ -7,7 +7,7 @@ browser-compat: html.elements.h1
 {{HTMLSidebar}}
 
-The **`<h1>`** to **`<h6>`** [HTML](/en-US/docs/Web/HTML) elements represent six levels of section headings. `<h1>` is the highest section level and `<h6>` is the lowest.
+The **`<h1>`** to **`<h6>`** [HTML](/en-US/docs/Web/HTML) elements represent six levels of section headings. `<h1>` is the highest section level and `<h6>` is the lowest. All heading elements are block-level by default.
```
suggesting a rephrase and adding some more info:

```suggestion
The **`<h1>`** to **`<h6>`** [HTML](/en-US/docs/Web/HTML) elements represent six levels of section headings. `<h1>` is the highest section level and `<h6>` is the lowest. By default, all heading elements create a [block-level ](/en-US/docs/Glossary/Block-level_content) box in the layout, starting on a new line and taking up the full width available in their containing block.
```

Arisch24 replied:
Noted with that @dipikabh. I will get on it right away.

dipikabh reviewed (changes requested):
Hello @Arisch24! Thanks for adding the clarifying text to fix the doc issue. I've suggested a rephrase and addition of a link to our glossary.

Arisch24 committed:
Rephrasing the sentence

More detailed explanation and an additional link to block-level related content

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

Arisch24 committed:
Remove spacing

Linter doesn't accept spacing in links

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

dipikabh reviewed (approved):
Thank you, @Arisch24! Congrats on landing your first PR.

Welcome to the MDN project 🎉

Arisch24 commented:
Your welcome @dipikabh and thank you for the guidance.
